fineract-provider/src/main/java/org/apache/fineract/infrastructure/creditbureau/service/CreditReportWritePlatformServiceImpl.java

```diff
@@ -146,20 +140,24 @@ public String addCreditReport(Long bureauId, File creditReport, FormDataContentD
 
     }
 
-    private Optional<String> getCreditBureau(Long creditBureauID) {
+    private String getCreditBureau(Long creditBureauID) {
```

Comment:
Let's rename the method. It's not returning a CreditBureau object but only its name. Should be `getCreditBureauName`